### PR TITLE
Bump freedv-gui to 1.6.0.

### PR DESCRIPTION
**Pull request by tmiw**
Bumps FreeDV to 1.6.0 and removes "devel" suffix. Additionally, this creates a "make dist" target to generate a tarball per #150.

### CODE REVIEW

File: USER_MANUAL.md [hunk 1]
```diff
@@ -724,6 +724,33 @@ LDPC | Low Density Parity Check Codes - a family of powerful FEC codes
 
 # Release Notes
 
+## V1.6.0 August 2021
+
+1. Bugfixes:
```
Comment (tmiw) [inline USER_MANUAL.md:729]: @drowe67, did I miss anything here?

**Comment by tmiw** [comment]
Additionally, is 1.6.0 reasonable or should it be something else?

**Comment by drowe67** [comment]
Looks good Mooneer - you've done lots of work!

File: USER_MANUAL.md [hunk 2]
```diff
@@ -724,6 +724,33 @@ LDPC | Low Density Parity Check Codes - a family of powerful FEC codes
 
 # Release Notes
 
+## V1.6.0 August 2021
+
+1. Bugfixes: 
+    * Suppressed clipping of TX speech when PTT is released. (PR #123)
+    * Added missing mode labels for 800XA and 2400B as a result of implementing multi-RX in 1.5.3. (PR #128)
+    * Fixed analog passthrough when using 2400B. (PR #130)
+    * Fixed non-responsible scroll controls on macOS. (PR #139)
```
[the remark this answers is not in the document]
Comment (tmiw) [reply USER_MANUAL.md]: Oops, fixed, thanks!

**Comment by drowe67** [comment]
@tmiw - I'll merge this now.  would you also like to add a tag once it's in master?

**Comment by tmiw** [comment]
@drowe67, see https://github.com/drowe67/freedv-gui/releases/tag/v1.6.0. I'll upload the tarball and binaries there tonight.

**Comment by tmiw** [comment]
Hmm, got this error: https://github.com/drowe67/freedv-gui/runs/3257191655?check_suite_focus=true. Any ideas?

**Pull request merged**